tests/integration/tools/client/src/main/java/io/helidon/tests/integration/tools/client/HelidonProcessRunner.java

```diff
@@ -0,0 +1,411 @@
+/*
+ * Copyright (c)  2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.tests.integration.tools.client;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.net.SocketAddress;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import io.netty.util.internal.SocketUtils;
+
+public class HelidonProcessRunner {
+    private static final Logger LOGGER = Logger.getLogger(HelidonProcessRunner.class.getName());
+
+    public static int HTTP_PORT = 8080;
```

Comment:
This should not be static. Also we should never run on port `8080`.

Reply:
This is workaround until we'll implement jUnit constructors with HTTP port. It's set to currently used port after process is started. You once asked me to put this static variable into HelidonProcessRunner.
It will be removed later.

Reply:
Changed initial value to -1 to make sure 8080 is not used.